src/components/markdown-input.less

```diff
@@ -39,6 +39,7 @@
 }
 
 .MarkdownInput-preview {
+  overflow-x: scroll;
```

Comment:
`scroll` shows permanent scrollbars on most systems, `auto` would be better.

Reply:
Fixed with another CSS approach, thanks!